src/Datasource/Dummy.php

```diff
@@ -0,0 +1,164 @@
+<?php
+declare(strict_types=1);
+
+/**
+ * CakePHP(tm) : Rapid Development Framework (https://cakephp.org)
+ * Copyright (c) Cake Software Foundation, Inc. (https://cakefoundation.org)
+ *
+ * Licensed under The MIT License
+ * For full copyright and license information, please see the LICENSE.txt
+ * Redistributions of files must retain the above copyright notice.
+ *
+ * @copyright     Copyright (c) Cake Software Foundation, Inc. (https://cakefoundation.org)
+ * @link          https://cakephp.org CakePHP(tm) Project
+ * @since         2.1.0
+ * @license       https://opensource.org/licenses/mit-license.php MIT License
+ */
+namespace Cake\ElasticSearch\Datasource;
+
+use Cake\Database\Driver;
+
+/**
+ * Represents a dummy driver used only for convertions
+ * between types of ElasticSearch and PHP.
+ */
+class Dummy extends Driver
+{
+    /**
+     * Establishes a connection to the database server.
+     *
+     * @return bool True on success, false on failure.
+     */
+    public function connect()
```

Comment:
All of these can `@inheritDoc`

Reply:
I tried that, but codesniffer triggered error.

Should I ignore it (cs error)?

Reply:
I have no idea without seeing the error.

Reply:
My fault, I had put `{@inheritDoc}` instead of `@inheritDoc`. Fixed now.